schemas/general/SCHEMA.md

```diff
@@ -0,0 +1,70 @@
+# General Schema
+
+Table definitions for the generic schema. These tables can be used for any protocol.
+
+## Schema
+
+### Incentive Claim Data
+
+| Property              | Description                                                     | Type   |
```

Comment:
to add in transaction_id and evt_index

Reply:
there is a possibility whereby users can just do multiple claims over time

Reply:
Added.

That doesn't change the schema though, correct?

Reply:
yes, having the mentioned 2 columns above is good